dev/README.md

```diff
@@ -35,29 +35,47 @@ Currently successfully brings up charts - no guarantee that everything is workin
 
     Example result:
     ```
-    "CrDr5orsnFLg3SL3Jpk9cH0wXENAWqbY"
+    "BxTc70FW7gL5MEueAFcxGE08nbWWsytE"
     ```
 2. query an authenticated endpoint (fill out the `session_token` from above)
     ```
-    session_token="CrDr5orsnFLg3SL3Jpk9cH0wXENAWqbY"
-    curl -k 'https://localhost:8080/graphql' -H 'Content-Type: application/json' -H 'Accept: application/json' -H "Authorization: bearer ${session_token}" --data-binary '{"query": "{ me { phone } }" }'
+    session_token="BxTc70FW7gL5MEueAFcxGE08nbWWsytE"
+    curl -k 'http://localhost:4002/graphql' -H 'Content-Type: application/json' -H 'Accept: application/json' -H "Authorization: bearer ${session_token}" --data-binary '{"query": "{ me { phone } }" }'
     ```
     Expected result:
     ```
-    {"data":{"me":{"phone":"+59981730222"}}}
+    {"data":{"me":{"phone":"+16505554321"}}}
     ```
 #### Test the graphql-admin api
 
 1. get session token
     ```
-    curl -ksS 'https://localhost:8080/admin/graphql' -H 'Content-Type: application/json' -H 'Accept: application/json' --data-binary '{"query":"mutation login($input: UserLoginInput!) { userLogin(input: $input) { authToken } }","variables":{"input":{"phone":"+59981730222","code":"111111"}}}' | jq '.data.userLogin.authToken'
+    curl -ksS 'http://localhost:4002/admin/graphql' -H 'Content-Type: application/json' -H 'Accept: application/json' --data-binary '{"query":"mutation login($input: UserLoginInput!) { userLogin(input: $input) { authToken } }","variables":{"input":{"phone":"+16505554321","code":"321321"}}}' | jq '.data.userLogin.authToken'
```

Comment:
will need to change the phone number and code in the sample request above as well https://github.com/GaloyMoney/charts/pull/2538/files#diff-e14025c0fa40d4857e4b40fc96ea5ee995afe300014626b68cb55a479fa5b8fcR31